dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipTypeRestRepositoryIT.java

```diff
@@ -0,0 +1,254 @@
+        String leftTypeString = "Person";
+        String rightTypeString = "Project";
+        String leftLabel = "isProjectOfPerson";
+        String rightLabel = "isPersonOfProject";
+        checkRelationshipType(leftTypeString, rightTypeString, leftLabel, rightLabel);
+    }
+
+    @Test
+    public void findPersonOrgUnitRelationshipType() throws SQLException {
+        String leftTypeString = "Person";
+        String rightTypeString = "OrgUnit";
+        String leftLabel = "isOrgUnitOfPerson";
+        String rightLabel = "isPersonOfOrgUnit";
+        checkRelationshipType(leftTypeString, rightTypeString, leftLabel, rightLabel);
+    }
+
+    @Test
+    public void findProjectOrgUnitRelationshipType() throws SQLException {
+        String leftTypeString = "Project";
+        String rightTypeString = "OrgUnit";
+        String leftLabel = "isOrgUnitOfProject";
+        String rightLabel = "isProjectOfOrgUnit";
+        checkRelationshipType(leftTypeString, rightTypeString, leftLabel, rightLabel);
+    }
+
+    @Test
+    public void findJournalJournalVolumeRelationshipType() throws SQLException {
+        String leftTypeString = "Journal";
+        String rightTypeString = "JournalVolume";
+        String leftLabel = "isVolumeOfJournal";
+        String rightLabel = "isJournalOfVolume";
+        checkRelationshipType(leftTypeString, rightTypeString, leftLabel, rightLabel);
+    }
+
+    @Test
+    public void findJournalVolumeJournalIssueRelationshipType() throws SQLException {
+        String leftTypeString = "JournalVolume";
+        String rightTypeString = "JournalIssue";
+        String leftLabel = "isIssueOfJournalVolume";
+        String rightLabel = "isJournalVolumeOfIssue";
+        checkRelationshipType(leftTypeString, rightTypeString, leftLabel, rightLabel);
+    }
+
+    private void checkRelationshipType(String leftType, String rightType, String leftLabel, String rightLabel)
+        throws SQLException {
+        RelationshipType relationshipType = relationshipTypeService
+            .findbyTypesAndLabels(context, entityTypeService.findByEntityType(context, leftType),
+                                  entityTypeService.findByEntityType(context, rightType),
+                                  leftLabel, rightLabel);
+        assertNotNull(relationshipType);
+        assertEquals(entityTypeService.findByEntityType(context, leftType),
+                     relationshipType.getLeftType());
+        assertEquals(entityTypeService.findByEntityType(context, rightType),
+                     relationshipType.getRightType());
+        assertEquals(leftLabel, relationshipType.getLeftLabel());
+        assertEquals(rightLabel, relationshipType.getRightLabel());
+    }
+
+    @Test
+    public void getAllRelationshipTypesEndpointTest() throws Exception {
+        //When we call this facets endpoint
+        List<RelationshipType> relationshipTypes = relationshipTypeService.findAll(context);
```

Comment:
it is better to use hardcoded values for the test otherwise error in the findAll will be hidden to the test

Reply:
I don't see this as a high priority fix since any errors to `findAll` should be found in the `findAllRelationshipTypesTest()` (which is currently also in this same IT file, but @abollini noted should be moved to a separate Unit Test file).  So, using real data here seems fine to me, there's no reason to require hardcoded values.

Reply:
I prefer to have highly independent test so that we don't need to assume that something is verified elsewhere. This is especially useful now that we get IT for everything over a low unit test coverage code base. Having different approach in the codebase can also lead other developer to pick the "wrong" example. If you agree about which is the preferred approach please add a //FIXME - ticket to rework on that

Reply:
As the RelationshipTypes being provided out-of-the-box are not finalized, I'd recommend we add this request to the open ticket around RelationshipTypes: https://jira.duraspace.org/browse/DS-4184  I've added it to that ticket description